input/pages/measure-terminology-service.md

```diff
@@ -539,6 +563,36 @@ The full example is available here:
 
 * [eCQM Release, 2020-05-07](Library-ecqm-update-2020-05-07.html)
 
+#### Discovery of Expansion Identifiers
+
+To discover what values of the $expand `expansion` parameter are supported by the Measure Terminology Service, start with a Library search using the `_profile` parameter to limit the search to [CQFMQualityProgram](StructureDefinition-quality-program-cqfm.html), and the `contained-parameter-name` search parameter with the value `expansion`.
```

Comment:
We should add a paragraph describing support for the use of expansion identifiers:

Note that some terminology servers support the use of persisted expansions to ensure stable expansion of value sets included as part of a release. This approach provides an alternative to explicitly stating binding versions for all the unreferenced canonical references used in the artifacts included in a release. Because expansion identifiers are server-specific, the preferred approach is to make use of the version manifest to state binding versions. However, the use of expansion identifiers MAY be used as an alternative.